index.js

```diff
@@ -153,8 +153,13 @@ Parser.prototype._transform = function(file, encoding, done) {
     // finally we add the parsed keys to the catalog
     // =============================================
     for (var j in keys) {
+        var key = keys[j];
         // remove the backslash from escaped quotes
-        var key = keys[j].replace(/\\('|"|`)/g, '$1');
+        key = key.replace(/\\('|"|`)/g, '$1')
+        key = key.replace(/\\n/g, '\n');
```

Comment:
`\n` is a single char, so it `\r` and `\t`. You cannot match on the `\` part because it only exists in the pre-parsed source.